examples/examples-log4j/src/main/proto/helloworld/helloworld.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2015 The gRPC Authors
```

Comment:
This file is identical to .../examples/src/main/proto/helloworld.proto. I understand the examples directory and package structure makes this necessary but can we think of eliminating this duplication? Also that one is under .../proto but this one is under .../proto/helloworld . Wonder why the difference.